### ISSUE TITLE
Clang/C2 Status

### ISSUE THREAD

**Issue by LukeMauldin** [microsoft/CMake#18]
What is status of getting Clang/C2 merged into master of the fork and then pushed back upstream to kitware/cmake?

**Comment by khouzam** [comment]
The support has been done in parallel and is currently in the nightly builds of CMake. I'm guessing it will be included in the next major release.

We will be working on their code going forward.
